pytest.ini

```diff
@@ -0,0 +1,3 @@
+[pytest]
+markers =
+    web: internet required to run this test (deselect with '-m \"not web\"')
```

Comment:
Out of curiosity, for the tests that require an internet connect, would the alternative been to download those files and monkeypatch them?

Reply:
yes, a (possibly better) solution would be to monkeypatch the bits that require conda-vendor and also our download functions. For now, its sufficient to let it reach out for the tests while we focus our attention on more important aspect of the codebase.